ja/newsletters.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: newsletters
+lang: ja
+title: Newsletters-ja
+name: newsletters-ja
+permalink: /ja/newsletters/
+share: false
+version: 1
+---
+
+Would you like to help translate our newsletters? See the [CONTRIBUTING
+documentation](https://github.com/bitcoinops/bitcoinops.github.io/blob/master/CONTRIBUTING.md)
+and the [Japanese translation issues and
+PRs](https://github.com/bitcoinops/bitcoinops.github.io/pulls?&q=label%3Alocalization-japanese)
+in our github repo.
+{:.center}
```

Comment:
Not a blocker, but I think this might look better in italics (and possibly not centered).

Reply:
done!